src/Login.css

```diff
@@ -0,0 +1,63 @@
+.app {
+    font-family: sans-serif;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    gap: 20px;
+    height: 100vh;
+    font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+    background-color: #f8f9fd;
+}
+
+input[type="text"],
```

Comment:
We want this file and Login.tsx to live together in a subfolder called 'Login' in the 'Pages' folder:

src->Pages->Login-> {css and tsx file)